src/Oscoin/Configuration.hs

```diff
@@ -0,0 +1,11 @@
+{-- | Umbrella module which gathers all the types which, when combined,
+--- determines the particular configuration an Oscoin node is running with.
+--}
```

Comment:
The `{}` is not necessary here - same comment for the other module headers :jack_o_lantern:

Reply:
Old habits!

Reply:
ps. It shouldn't matter too much, as @kim's PR [here](https://github.com/oscoin/oscoin/pull/459) introduces essentially the same module. I'll tell you what, I am going to merge the aforementioned PR (as it has 2 green reviews) and gonna deal with the ~pain~ conflicts 😅